src/App.jsx

```diff
@@ -1,55 +1,10 @@
-import logo from './logo.svg';
-import { Counter } from './features/counter/Counter';
 import './App.css';
+import Contactus from './containers/Contactus';
```

Comment:
The folder name should be "pages" not "containers". It is not wrong to call it containers at all but "pages" is the naming convention we go with.